js/arena.js

```diff
@@ -0,0 +1,36 @@
+import Gladiator from './gladiator.js';
+
+export default class Arena {
+    constructor(name) {
+        this.name = name;
+        this.gladiators = [];
+    }
+    addGladiator(gladiators) {
+        if (this.gladiators.length < 2) {
+            this.gladiators.unshift(gladiators);
+
+        }
+    }
+
+
+    fight() {
+        if (this.gladiators.length === 2) {
+            const fight1 = this.gladiators[0].weapon;
+            const fight2 = this.gladiators[1].weapon;
+
+        } else if ((fight1 === "Trident" && fight2 === "Spear") ||
+            (fight1 === "Spear" && fight2 === "Club") ||
+            (fight1 === "Club" && fight2 === "Trident")) {
+            console.log(fight1 + 'win');
+            fight2.pop();
```

Comment:
.pop() does not work here. pop is a method that only works on an array but not on an element.

one way to remove an element is to use a method called .splice()
here is a link on how to use it
[splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)

or smart use of pop() and shift() will solve this problem